CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## 17.3.0 [#578](https://github.com/openfisca/openfisca-core/pull/578)
+
+#### New features
+
+- Improve the representations of parameters when navigating the legislation in Python.
+
+For instance:
+```
+tax_benefit_system.parameters.benefits
+
+>>> basic_income:
+>>>   2015-12-01: 600.0
+>>> housing_allowance:
+>>>   2016-12-01: None
+>>>   2010-01-01: 0.25
+```
+
+- Be more flexible about parameters definitions
+
+The two following expressions are for instance striclty equivalent:
+
+```
+Parameter("taux", {"2015-01-01": 2000})
```

Comment:
From what I understand, `"taux"` represent a specific parameter. If so, could we find a better representation such as `"income.tax.rate"` ?